master.yaml

```diff
@@ -2,49 +2,80 @@
 # mayaonline can parse this yaml to obtain tests specific to applications (based on tag) & populate test params,
 # fulfill pre-requisites etc, before deployment. The params are explicitly typed to prompt right values from user.
 # Test parameters can be of different kinds such as ENV, configmap etc., with their own properties.
-# '@userdata' is a placeholder which is either autofilled/userfilled. If empty, use defaults. The test has tags which
-# will help with categorization & application-mapping
+# The test has tags which will help with categorization & application-mapping.
 
 
 tests:
-  - name: pod_kill
-    test_tags: ['mysql', 'pumba']
-    specPath: ./apps/percona/tests/mysql_data_persistence/run_litmus_test.yaml
+  - name: openebs-volume-replica-failure
+    test_tags: ['mysql', 'openebs']
```

Comment:
In this line `openebs` tag refers to what? @ksatchit

Reply:
This test uses openebs chaos @a4abhishek . As per my understanding, the tags may reflect key elements of the test metadata (application: mysql, chaostype: openebs). If it were app_tags, then it would have been restricted to mysql.

Reply:
Changed to app_tags instead of test_tags, with mysql the current app supported